sweetest/src/main/java/com/mysugr/sweetest/framework/dependency/DependencyManager.kt

```diff
@@ -20,6 +20,10 @@ class DependencyManager(setupHandlerReceiver: (DependencySetupHandler) -> Unit)
     private val initializerContext = object : DependencyInitializerContext() {
         override fun <T : Any> instanceOf(clazz: KClass<T>): T {
             val configuration = configurations.getAssignableTo(clazz)
```

Comment:
as talked about f2f: please

* extract and unify behavior of `instanceOf` with `by dependency` 
* remove the auto-provision when neither a configuration nor state was found for the type requested

thx 🙂